### PR TITLE
Contact page section

### PR DESCRIPTION
**Pull request by DiamondNath07**
### Contact page with formspree
This page is built using the following technologies;
- Add index.html
- Add styles.css
- Add linters.yml
- Add formspree as a data source collection
 **All these were used for this contact page section**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -324,7 +324,7 @@ <h5>
       </article>
```
Comment (HAMMAS-SALEEM) [inline index.html:324]: - [ ] On line 317 replace the `section` tag with `footer` tag to follow the HTML Semantic tags :blush: due to SEO purpose :thinking:

File: index.html [hunk 2]
```diff
@@ -324,7 +324,7 @@ <h5>
...
       <form class="form" action="https://formspree.io/f/xzbwrooo" method="POST">
         <label>
-          <input name="text" maxlength="30" type="text" required placeholder="Full name" />
+          <input name="text" maxlength="30" minlength="15" type="text" required placeholder="Full name" />
         </label>
         <label>
           <input email="email" type="email" required value="Email address" />
```
Comment (HAMMAS-SALEEM) [inline index.html:330]: image

- [ ] Your forgot to add these social icons and these lines i guess :smile: but you can add it and make your design look similar to the Figma template to make it more professional :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @DiamondNath07 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following some requirements
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:330]: fixed.

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:324]: fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @DiamondNath07      , 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**